domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/stoptimes/TimepointStoptime.java

```diff
@@ -0,0 +1,37 @@
+package org.mobilitydata.gtfsvalidator.domain.entity.stoptimes;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class TimepointStoptime extends TimepointBase {
+
+    public TimepointStoptime(@NotNull String tripId,
+                             @NotNull String arrivalTime,
+                             @NotNull String departureTime,
+                             @NotNull String stopId,
+                             int stopSequence,
+                             @Nullable String stopHeadsign,
+                             @NotNull PickupType pickupType,
+                             @NotNull DropOffType dropOffType,
+                             @Nullable Float shapeDistTraveled) {
+        super(tripId, arrivalTime, departureTime, stopId, stopSequence, stopHeadsign, pickupType, dropOffType,
+                shapeDistTraveled);
+    }
+
+    public static class TimepointStoptimeBuilder extends TimepointBaseBuilder {
+
+        public TimepointStoptimeBuilder(@NotNull String tripId,
+                                        @NotNull String arrivalTime,
+                                        @NotNull String departureTime,
+                                        @NotNull String stopId,
+                                        int stopSequence) {
+            super(tripId, arrivalTime, departureTime, stopId, stopSequence);
+
+        }
+
+        public TimepointStoptime build() {
```

Comment:
Could that be in the base builder class as the only difference is with nullability of builder constructor parameter?

Reply:
If we do so, we might not be able to know if the entity is a TimepointStopTime or an ApproximatedTimeStopTime. This will be documented in javadocs for clarifications.